subjects/weareunique/README.md

```diff
@@ -0,0 +1,57 @@
+## weareunique
+
+### Instructions
+
+Write a function that takes two strings and returns the number of characters that are not included in both, without doubles.
```

Comment:
This a suggestion:  "...without repeated characters" instead of "...without doubles."

Reply:
agreed. Fixed